### PR TITLE
ON-7139 # Added draft service

### PR DESCRIPTION
**Pull request by mymattcarroll**

### CODE REVIEW

File: docs/draft-service.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+### `getDrafts()`
+
+Get an array of [Draft](#draft)s for the currently logged in user.
+
+```js
+const drafts = await draftService.getDrafts()
+```
+
+#### Draft
```
Comment (Zaxist) [inline docs/draft-service.md:41]: Can this go?

File: src/auth-service.js [hunk 2]
```diff
@@ -24,10 +25,9 @@ export {
   logout,
 }
 
-let authTenant = null
+let isInitialized = null
```
Comment (Zaxist) [inline src/auth-service.js:28]:
```suggestion
let isInitialized = false
```
?

File: docs/draft-service.md (hunk 1 above)
Comment (mymattcarroll) [reply docs/draft-service.md]: yep

File: src/draft-service.js [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+
+import _differenceBy from 'lodash.differenceby'
+import { v4 as uuidv4 } from 'uuid'
+import localForage from 'localforage'
+
+import OneBlinkAppsError from './services/errors/oneBlinkAppsError'
+import { isOffline } from './offline-service'
+import { getUsername, isLoggedIn } from './services/cognito'
+import { getIssuerFromJWT } from './auth-service'
+import { uploadDraftData, putDrafts } from './services/api/drafts'
```
Comment (Zaxist) [inline src/draft-service.js:12]:
```suggestion
import { uploadDraftData, putDrafts } from './services/api'
```

File: src/auth-service.js (hunk 2 above)
Comment (mymattcarroll) [reply src/auth-service.js:28]: This is being removed in a later pull request.

File: src/draft-service.js (hunk 3 above)
Comment (mymattcarroll) [reply src/draft-service.js:12]: I have removed these in a later pull request too. 👍  Was just more code to maintain.

**Pull request merged**